File: Javascript/1131.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var ng = 0;
var escolha = 0;
var empates = 0;
var vinter = 0;
var vgremio = 0;
while (escolha == 0) {
    var valores = lines.shift("XXX");
    var [inter, gremio] = valores.split(" ");
    inter = parseInt(inter);
    gremio = parseInt(gremio);
    if (inter>gremio) {
        vinter++;
    }
    if (inter<gremio) {
        vgremio++;
    }
    if (inter==gremio) {
        empates++;
    }
    console.log("Novo grenal (1-sim 2-nao)");
    numero = parseInt(lines.shift());
    ng++
    if (numero == 2) {
        escolha++;
    }
}
console.log(ng + " grenais");
console.log("Inter:"+vinter);
console.log("Gremio:"+vgremio);
console.log("Empates:"+empates);

if (vinter>vgremio) {
    console.log("Inter venceu mais");
}
else if (vinter<vgremio) {
    console.log("Gremio venceu mais");
}
else if (vinter==vgremio) {
    console.log("Nao houve vencedor")
}

File: Java/1131.java
import java.util.Scanner;

public class grenais {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        int grenais = 0;
        int empates = 0;
        int vitoriasInter = 0;
        int vitoriasGremio = 0;

        for (int i = 0; i < 1; i++) {
            int entradaInter = entrada.nextInt();
            int entradaGremio = entrada.nextInt();

            if(entradaInter>entradaGremio){
                vitoriasInter++;
            } else if(entradaInter<entradaGremio){
                vitoriasGremio++;
            } else {
                empates++;
            }

            System.out.println("Novo grenal (1-sim 2-nao)");
            grenais++;

            int opcao = entrada.nextInt();
            if(opcao == 1){
                i--;
            } else {
                i++;
            }
        }

        System.out.println(grenais + " grenais");
        System.out.println("Inter:" + vitoriasInter);
        System.out.println("Gremio:" + vitoriasGremio);
        System.out.println("Empates:" + empates);

        if(vitoriasInter>vitoriasGremio) {
            System.out.println("Inter venceu mais");
        }else if(vitoriasGremio>vitoriasInter){
            System.out.println("Gremio venceu mais");
        } else {
            System.out.println("Nao houve vencedor");
        }
    }
}
